main/webapp/modules/core/scripts/views/data-table/add-q-column-dialog.html

```diff
@@ -1,4 +1,5 @@
 <div class="dialog-frame" style="width: 700px;">
+<form action="javascript:okButton;">
```

Comment:
I'd prefer using jQuery's submit method to bind the event to the form, rather than an explicit action (since that relies on global naming).